lib/index.js

```diff
@@ -13,5 +13,38 @@ module.exports = (dirname, manifestExtras) => {
 
     const manifest = Manifest.concat(manifestExtras || []);
 
-    return Haute(dirname, manifest);
+    return (server, options, next) => {
+
+        if (typeof options === 'function' ||
```

Comment:
So as not to repeat `typeof next === 'function'`, does it make more sense for this block to be structured as,
```js
if (typeof next === 'function') {
  //
}
else if (typeof options === 'function') {
  //
}
```
?